ui/js/directives/treeherder/clonejobs.js

```diff
@@ -81,21 +75,26 @@ treeherder.directive('thCloneJobs', [
             };
         }
 
-        jobs = $(selector);
-        var idx = jobs.index(jobs.filter(".selected-job"));
-        idx = getIndex(idx, jobs);
+        jobs = $(jobNavSelector.selector).filter(":visible, .selected-job");
```

Comment:
This was the fix to the last time we pushed it.  I needed to include ``.selected-job`` in this filter so that it was included in the search on line 80.  Without it, after classifying, the selected job was no longer visible, therefore not matching this filter.  Then on line 80, in didn't find ``.selected-job`` within the filtered list and just resorted to selecting the first unclassified failure.